stopcovid/sms/enqueue_outbound_sms.py

```diff
@@ -42,62 +39,68 @@ class OutboundSMS:
     media_url: Optional[str] = None
 
 
-def get_localized_messages(
-    dialog_event: DialogEvent, messages: List[PromptMessage], **kwargs
-) -> List[OutboundSMS]:
-    language = dialog_event.user_profile.language
-
-    additional_args = {
-        "company": dialog_event.user_profile.account_info.get("employer_name", "your company"),
-        "name": "",
-    }
-    if dialog_event.user_profile.name is not None:
-        additional_args["name"] = dialog_event.user_profile.name.split(" ")[0]
-    additional_args.update(kwargs)
+def get_messages(dialog_event: DialogEvent, messages: List[PromptMessage],) -> List[OutboundSMS]:
 
     return [
         OutboundSMS(
             event_id=dialog_event.event_id,
             phone_number=dialog_event.phone_number,
-            body=localize(message.text, language, **additional_args)
```

Comment:
All messages are already fully localized _besides_ the three dynamic prompt responses that this emits